go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/coreos/ignition/v2 v2.10.1
 	github.com/go-openapi/runtime v0.19.28
 	github.com/go-openapi/strfmt v0.20.1
+	github.com/go-openapi/swag v0.19.9
```

Comment:
Is adding swag necessary? or just simple de-referencing  can do

Reply:
i think we will need it in any case for unitests, not sure why not to use it and make test code more complex